src/app/authentication/models/signUpModel.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+type User struct {
+	Name        string `json:"name" validate:"required" gorm:"column:name"`
+	Email       string `json:"email" validate:"required,email" gorm:"column:email"`
+	PhoneNumber int    `json:"phoneNumber" validate:"required" gorm:"column:phoneNumber"`
```

Comment:
Change phone number datatype to unint64 and add all other validation other than required